apps/drain-safe/src/App.tsx

```diff
@@ -173,19 +193,13 @@ const App: React.FC = () => {
           <TextField onChange={onToAddressChange} value={toAddress} label="Recipient" />
```

Comment:
nice, remember that `TextField` already handles errors. For this case isn't neccessary because you are handling a global error. 
https://safe-components.dev.gnosisdev.com/?path=/docs/inputs-textfield--simple-text-field#error

Reply:
For this you might need latest version of safe-react-components as I mentioned above